File: synonyms-java/src/main/java/org/ygrene/demos/synonyms/SynonymsApplication.java
package org.ygrene.demos.synonyms;

import org.springframework.boot.*;
import org.springframework.boot.autoconfigure.*;
import org.springframework.data.neo4j.repository.config.*;

@SpringBootApplication
@EnableNeo4jRepositories(basePackages = "org.ygrene.demos.synonyms.dal.repositories.neo4j")
public class SynonymsApplication {

    public static void main(String[] args) {
        SpringApplication.run(SynonymsApplication.class, args);
    }
}

File: synonyms-kotlin/src/main/kotlin/org/ygrene/demos/synonyms/SynonymsApplication.kt
package org.ygrene.demos.synonyms

import org.springframework.boot.SpringApplication
import org.springframework.boot.autoconfigure.SpringBootApplication


@SpringBootApplication
class SynonymsApplication

fun main(args: Array<String>) {
    SpringApplication.run(SynonymsApplication::class.java, *args)
}
